src/go/rpk/pkg/cli/cluster/partitions/toggle.go

```diff
@@ -204,40 +200,3 @@ func runToggle(ctx context.Context, cl *adminapi.AdminAPI, all bool, topicArg, p
 	}
 	return g.Wait()
 }
-
```

Comment:
This was moved to the `out` package. The `toggle_test.go` file was deleted and moved to `in_test.go` (inside `out` package)